### PR TITLE
Feature: ephemeral keys

### PR DESCRIPTION
**Pull request by SignSpice**
Not sure this is the best way to go about implementing this feature - I am new to Clojure. 

I was unable to find the official documentation for this on the stripe website, doesn't seem to exist. Although it is mentioned in some of the guides, such as this one: https://stripe.com/docs/payments/accept-a-payment?platform=react-native#react-native-add-server-endpoint

### CODE REVIEW

File: src/magnet/payments/stripe/ephemeral_key.clj [hunk 1]
```diff
@@ -0,0 +1,19 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+(ns magnet.payments.stripe.ephemeral-key
+  (:require [magnet.payments.core :as core]
+            [magnet.payments.stripe.core :refer [execute]])
+  (:import [magnet.payments.stripe.core Stripe]))
+
+(do (def ^:const api-definition
```
Comment (lucassousaf) [inline src/magnet/payments/stripe/ephemeral_key.clj]: Hi @SignSpice ! Thanks a lot for the pull request! I just have one comment, maybe it is a mistake but wrapping the `(def)` and the `(extend-protocol)` in a `do` is not necessary.

**Comment by lucassousaf** [comment]
Hi @SignSpice,
Thank you very much for your work!
I left one comment on your changes. Once it is fixed, before merging the changes, we need you to first sign a Contribution License Agreement following our [contributing guideline](https://github.com/magnetcoop/.github/blob/befa53fa59124f5dc37bdf72718a9d866cd08971/CONTRIBUTING.md) 
Thanks in advance!

**Comment by SignSpice** [comment]
@lucassousaf You're welcome! 

I was not able to find the comment you left.  Could you send me a link or post it here?

File: src/magnet/payments/stripe/ephemeral_key.clj (hunk 1 above)
Comment (SignSpice) [reply src/magnet/payments/stripe/ephemeral_key.clj]: Oops, yep that was a mistake. Left over from moving forms between files.

**Comment by SignSpice** [comment]
@lucassousaf Do you have any insight on the failing build on travis? All the tests pass when I run them locally. But the travis build doesn't even get so far as running the tests, if I understand it correctly.

**Comment by lucassousaf** [comment]
Hello @SignSpice, it seems that the problem with the Travis CI failing was due to a change in the OpenJDK 8 image, and it affects old versions of `http-kit`. So, I upgraded the library version (just pushed the changes to the main branch) to the latest and the CI is working again.

**Comment by SignSpice** [comment]
@lucassousaf Okay. Is there another option for submitting the signed license agreement?  I am on Linux, and the online version of word doesn't work with the form inputs, and LibreOffice doesn't seem to let me submit the form either.

**Comment by SignSpice** [comment]
Okay, now the build is running the tests, but the ephemeral keys test is failing because I set the test to try to pull the stripe api verison out of an environment variable.  So the build config would need to be changed for it to pass as is.  I could set the test to a hard-coded default too, I suppose. IE. 
`(or (System/getenv "STRIPE_API_VERSION") "2020-08-27")`

Does that sound good to you?

**Comment by lucassousaf** [comment]
> Okay, now the build is running the tests, but the ephemeral keys test is failing because I set the test to try to pull the stripe api verison out of an environment variable. So the build config would need to be changed for it to pass as is. I could set the test to a hard-coded default too, I suppose. IE. `(or (System/getenv "STRIPE_API_VERSION") "2020-08-27")`
> 
> Does that sound good to you?

Okay! I added the `STRIPE_API_VERSION` in the build config. So we won't need that hardcoded API version.

**Comment by lucassousaf** [comment]
> @lucassousaf Okay. Is there another option for submitting the signed license agreement? I am on Linux, and the online version of word doesn't work with the form inputs, and LibreOffice doesn't seem to let me submit the form either.

@SignSpice, yes, you can follow the instruction of the CLA assistant above. No need for submitting the files. We'll eventually update the contribution guidelines to address this issue. Thank you!

**Pull request merged**